fusion-cli/build/dev-runtime.js

```diff
@@ -140,6 +142,21 @@ module.exports.DevelopmentRuntime = function(
       }
     `;
 
+    /**
+     * An escape hatch to not respawn the server if the server bundle never changed.
+     * This saves substantial times for CSR apps, who rarely change code in their server bundle.
+     */
+    if (existsSync(entryFile)) {
```

Comment:
Nit. Good practice is to avoid using sync file operations. util/promisify can be used to wrap callback based functions.